Add kind image v1.21.2 for v1a4 tests

kashifest opened the pull request:
Since capm3 v1a4 tests can only survive with k8s < v1.22 we also need to prepull kind docker  image version 1.21.2

fmuyassarov commented on ci/scripts/image_scripts/source_cluster_container_images.sh:
```diff
@@ -10,7 +10,10 @@ export IMG_CENTOS_IMG="${IMG_CENTOS_IMG:-"docker.io/centos:centos8"}"
+    export KIND_NODE_IMAGE_VERSION=${KIND_NODE_IMAGE_VERSION:-"v1.22.0"}
+    # Since capm3 v1a4 tests can only survive with k8s < v1.22
+    # the following docker image also needs to be a part of the disk image.
+    export KIND_NODE_IMAGE_VERSION_CAPM3_v1a4="v1.21.2"   
```
You need to have IMG prefix, otherwise image won't be downloaded

kashifest replied:
Whatt??? how is it downloading the previous one? I thought havin IMG in the name is enough

kashifest replied:
Done

fmuyassarov replied:
We are downloading images that has IMG prefix in a loop  in https://github.com/Nordix/airship-dev-tools/blob/69c723b23c50c24ba1d039a8d7cce486eca05899/ci/scripts/image_scripts/source_cluster_container_images.sh#L17

fmuyassarov reviewed (changes requested):
Sorry, I meant

`export IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4="${IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4:-"kindest/node:v1.21.2"}" 
`

fmuyassarov commented:
> Sorry, I meant
> 
> `export IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4="${IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4:-"kindest/node:v1.21.2"}" `

and probably some shorter name would be better

kashifest commented:
> Sorry, I meant
> 
> `export IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4="${IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4:-"kindest/node:v1.21.2"}" `

I am not getting you, I have already added it in the name and I am not defaulting it

kashifest commented:
> > Sorry, I meant
> > `export IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4="${IMG_KIND_NODE_IMAGE_VERSION_CAPM3_v1a4:-"kindest/node:v1.21.2"}" `
> 
> I am not getting you, I have already added it in the name and I am not defaulting it

Got you , fixed
